packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -164,17 +164,29 @@ export default class RestApi {
 
     api
       .get('/backups', (req, res) => {
-        sendObjects([{ id: 'logs' }], req, res)
+        sendObjects([{ id: 'logs' }, { id: 'restore' }], req, res)
       })
       .get(
         '/backups/logs',
         wrap(async (req, res) => {
-          const logs = await app.getBackupNgLogsSorted({})
+          const logs = await app.getBackupNgLogsSorted({
+            filter: ({ message: m }) => m === 'backup' || m === 'metadata',
+          })
+          sendObjects(logs, req, res)
+        })
+      )
+      .get('/backups/restore', (req, res) => {
+        sendObjects([{ id: 'logs' }], req, res)
+      })
+      .get(
+        '/backups/restore/logs',
```

Comment:
what would you think of using 
`backups/logs/:id` and `restore/logs/:id` ?

That way , when (if?) we'll implement restore/backup action api they'll be namespaced like `backup/actions` and `restore/actions`

Reply:
I feel like `restore` is a backup feature, that's why I put it underneath, but I have no strong opinion on this.

I'll change it, and it probably change again in the future.